src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/builders/SerializedFormBuilder.java

```diff
@@ -400,14 +401,14 @@ public void buildFieldSerializationOverview(TypeElement typeElement, Content cla
      */
     protected void buildSerializableFields(Content target)
             throws DocletException {
-        SortedSet<VariableElement> members = utils.serializableFields(currentTypeElement);
-        if (!members.isEmpty()) {
+        Iterator<VariableElement> members = utils.serializableFields(currentTypeElement).iterator();
+        if (members.hasNext()) {
```

Comment:
Same comment as above

Reply:
I updated those two locations you mentioned, but decided not to static import `Objects.equals`: see e8d07a12fa2c3ac7a5a21edf28ecb40e685603ac.